dist/kafka-sampler/config.yaml

```diff
@@ -13,16 +13,23 @@ kafka:
   # Convert field names to all lowercase and divide nested structs with `_`
   # sarama:  (...)
 
-  # If unset, it will create a `Sampler` per each topic found in the Kafka cluster,
-  # supports regex RE2 syntax as described at https://github.com/google/re2/wiki/Syntax, except for `\C`.
-  # It always ignores internal topics like: `__consumer_offsets` and `__transaction_state`.
-  # topicfilter: 
+  # Topics contains all the configuration about the topics that have to be read from
```

Comment:
Mh I feel like there are too many `topics` in this sentence. No other top-level key has a description either. We can just remove it and only describe the sub-keys.

Reply:
Removed the comment